pom.xml

```diff
@@ -59,19 +64,67 @@
                     <groupId>org.apache.hadoop</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
+		<exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.module</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-mapreduce-client-core</artifactId>
-            <version>2.7.2</version>
+            <version>2.8.2</version>
             <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.module</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-common</artifactId>
-            <version>2.7.2</version>
+            <version>2.8.2</version>
             <scope>provided</scope>
+	    <exclusions>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.module</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+	<dependency> 
+	    <groupId>org.apache.hadoop</groupId> 
+	    <artifactId>hadoop-hdfs</artifactId> 
+	    <version>2.8.2</version> 
+	    <exclusions> 
+	        <exclusion> 
+	            <groupId>log4j</groupId> 
+	            <artifactId>log4j</artifactId> 
+	        </exclusion> 
+	        <exclusion> 
+	            <groupId>org.slf4j</groupId> 
+	            <artifactId>slf4j-log4j12</artifactId> 
+	        </exclusion> 
+	    </exclusions> 
+	</dependency> 
+        <dependency>
+            <groupId>org.datasyslab</groupId>
+            <artifactId>sernetcdf</artifactId>
```

Comment:
Is there a specific reason you move sernetcdf dependency to here? Note that, the sernetcdf dependency's scope should always be "provided". If you don't mention it, its scope is "compile".

"compile" will package sernetcdf to GeoSpark release package. This will lead to two problems:
(1) many version conflict issue. I believe you have seen that.
(2) the GeoSpark release jar will be very large because sernetcdf dependency itself is around 40-50 MB.

So if you just move sernetcdf to here or change its scope by mistake, please change it back to "provided"